src/main/java/yablunovskyi/bookstore/controller/BookController.java

```diff
@@ -33,6 +34,7 @@ public class BookController {
     @ResponseStatus(HttpStatus.OK)
     @Operation(summary = "Get all books",
             description = "Get a list of all available books from the datase")
+    @PreAuthorize("hasAnyRole('ADMIN', 'USER')")
```

Comment:
shouldn't admin user also have a role `USER`?

or you design your app in a way that admin does not have same priviliges as user has?

Reply:
Yes, ADMIN has the same priviliges as a USER, so to "findAll", "findById" methods I added @PreAuthorized("hasAnyRole('ADMIN', 'USER')") so ADMIN and USER can use this methods. And where only ADMIN has access to methods I add @PreAuthorize("hasRole('ADMIN')")